README.rst

```diff
@@ -50,6 +50,12 @@ See ``make help``.
 ``clean``
     No exotic dependencies are needed.
 
+``fixtures/debian``
+    The ``equivs`` and ``reprepro`` packages must be installed.
+
+``fixtures/debian-invalid``
+    Requires the same dependencies as the ``fixtures/debian`` target.
```

Comment:
```suggestion
    Depends on the ``fixtures/debian`` target.
```
For regular make users, this should imply that fixtures/debian is also built.

Reply:
And done.